### DISCUSSION TITLE
Table select from external source

### DISCUSSION THREAD

**Discussion by LigiaAb** [quasarframework/quasar#12924] [General - Components / Directives / etc] [answered]
I´m trying to set the selected row from an external source. 

I´m getting the full object that I want to select from a diferent component and trying to select this item in the q-table.

**Comment by MilosPaunovic** [comment, marked as answer]
You can populate array of selected items once you recieve your data from server.
[https://codepen.io/milospaunovic/pen/popRBez?editors=1011](https://codepen.io/milospaunovic/pen/popRBez?editors=1011)

**Reply by LigiaAb** [reply]
Thanks, that solved my problem
